apps/judicial-system/api/templates/ruling.pug

```diff
@@ -37,9 +37,9 @@ html
 
     h3 Úrskurðarorð
     if isolation
-      p Kærði, <strong>#{accusedName} kt. #{nationalId}</strong>, skal sæta gæsluvarðhaldi, þó ekki lengur en <strong>til #{custodyEndDay} #{custodyEndDate} kl. #{custodyEndTime}.</strong> Kærði skal <strong>sæta einangrun</strong> á meðan á gæsluvarðhaldinu stendur.
+      p Kærði, <strong>#{accusedName} kt. #{accusedNationalId}</strong>, skal sæta gæsluvarðhaldi, þó ekki lengur en <strong>til #{custodyEndDate} kl. #{custodyEndTime}.</strong> Kærði skal <strong>sæta einangrun</strong> á meðan á gæsluvarðhaldinu stendur.
```

Comment:
Custody restrictions are missing here. Example string is 

Kærði, Gunnar Apoi kt.1112902539 skal sæta gæsluvarðhaldi, þó ekki lengur en til 22. október 2020 kl. 12:31. Kærði skal sæta fjölmiðlabanni, heimsóknarbanni og einangrun á meðan á gæsluvarðhaldinu stendur.

Reply:
Ég skal laga það en væri til í að fá þessa PR í gegn fyrst. Þetta er work in progress. Ef ég breyti einhverju og pusha fer af stað endalaus keðja að buildum og mergjum sem tekur allan daginn:(